src/Umbraco.Core/Composing/ComponentCollection.cs

```diff
@@ -23,13 +23,13 @@ public ComponentCollection(Func<IEnumerable<IComponent>> items, IProfilingLogger
 
     public void Initialize()
     {
-        using (_profilingLogger.DebugDuration<ComponentCollection>(
+        using (_profilingLogger.IsEnabled(Logging.LogLevel.Debug) ? null : _profilingLogger.DebugDuration<ComponentCollection>(
```

Comment:
Am I reading this wrong: if `LogLevel.Debug` is enabled, don't log (`? null`) else, do log. Did you mean to negate the check? `!_profilingLogger.IsEnabled(Logging.LogLevel.Debug)`

Reply:
Makes sense. I assume I copied and pasted this. Fixed up.

Reply:
@nzdev Thanks! Same problem in 18 other files though! 😉 Anywhere you used a ternary, it's the wrong way around. 😅

Reply:
Ah wait, I see there's 2 commits, I only saw the last one, I think that covers all of them 👍